<launch>	

  <!-- Launch the usb camera acquisition node -->
  <node pkg="usb_cam" type="usb_cam_node" name="usb_cam_node_left" output="screen">      
	    <param name="video_device" value="/dev/video0" />
	    <param name="camera_name" value="left_camera" />
	    <param name="camera_frame_id" value="cam0" />
		<param name="image_width" value="1280" />
	    <param name="image_height" value="720" />
        <param name="framerate" value="30" />
		<param name="brightness" value="128" />
		<param name="contrast" value="150" />
		<param name="saturation" value="150" />
		<param name="sharpness" value="255" />
		<param name="gain" value="20" />
		<param name="exposure" type="int"  value="0" />
		<param name="absolute_exposure" type="int"  value="75" />
		<param name="autofocus" value="false"/>
		<param name="focus" type="int" value="0" />		
		<param name="pixel_format" type="string" value="mjpeg" />
  </node>

  <node pkg="usb_cam" type="usb_cam_node" name="usb_cam_node_right" output="screen">      
	    <param name="video_device" value="/dev/video1" />
	    <param name="camera_name" value="right_camera" />
	    <param name="camera_frame_id" value="cam0" />
		<param name="image_width" value="1280" />
	    <param name="image_height" value="720" />
        <param name="framerate" value="30" />
		<param name="brightness" value="128" />
		<param name="contrast" value="150" />
		<param name="saturation" value="150" />
		<param name="sharpness" value="255" />
		<param name="gain" value="20" />
		<param name="exposure" type="int"  value="0" />
		<param name="absolute_exposure" type="int"  value="75" />
		<param name="autofocus" value="false"/>
		<param name="focus" type="int" value="0" />		
		<param name="pixel_format" type="string" value="mjpeg" />
  </node>

</launch>
